src/domain/collection/curatedEventList/EventCards.tsx

```diff
@@ -16,14 +18,15 @@ const EventCards: React.FC<Props> = ({
   events,
   onShowMore,
   showMoreButton,
+  eventType,
 }) => {
   const { t } = useTranslation();
 
   return (
     <>
       <div className={styles.eventCards}>
         {events.map((event, id) => (
-          <LargeEventCard key={id} event={event} />
+          <LargeEventCard key={id} event={event} eventType={eventType} />
```

Comment:
strange that eventType was missing. I start to wonder now that do we event have to pass the type? Could we just detect the type from the event fields in some util function?

Reply:
Also wonder why is this added now, and was it a bug and some regression test missing?

Reply:
Nope, LargeEventCard had 'event' as default value. eventType is optional prop